sos/report/plugins/azure.py

```diff
@@ -8,8 +8,8 @@
 #
 # See the LICENSE file in the source distribution for further information.
 
-from sos.report.plugins import Plugin, UbuntuPlugin, RedHatPlugin
 import os
+from sos.report.plugins import Plugin, UbuntuPlugin, RedHatPlugin
```

Comment:
Why this change (repeated more times), please? I don't insist on either ordering of imports, just curious.

Reply:
For addressing [C0411 - wrong-import-order](https://pylint.readthedocs.io/en/latest/user_guide/messages/convention/wrong-import-order.html).

https://peps.python.org/pep-0008/#imports

> Imports should be grouped in the following order:
>
> 1. Standard library imports.
> 2. Related third party imports.
> 3. Local application/library specific imports.

Reply:
Ah I see, thanks for explanation.